aptos-move/framework/aptos-framework/sources/block.spec.move

```diff
@@ -52,9 +51,12 @@ spec aptos_framework::block {
         requires system_addresses::is_vm(vm);
         requires (proposer == @vm_reserved) ==> (timestamp::spec_now_microseconds() == timestamp);
         requires (proposer != @vm_reserved) ==> (timestamp::spec_now_microseconds() < timestamp);
-        requires event::counter(event_handle) == new_block_event.height;
 
         aborts_if false;
+
+        // While emitting a new block event, the number of them is equal to the current block height.
+        aborts_if event::counter(event_handle) != new_block_event.height;
```

Comment:
@junkil-park Should we use aborts_if or requires here?

Reply:
This line is not necessary because it's already required.

Reply:
Could you remove this before resolving this?